### PR TITLE
New WebView2 debug doc

### PR DESCRIPTION
**Pull request by JoshuaWeber**
Authored a new WebView2 doc for debugging WebView2.

### CODE REVIEW

File: microsoft-edge/webview2/howto/debug.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Right click on your project and select **Properties**.  Under **Configuration Properties**, select **Debugging** > **Debugger Type**, and select the **JavaScript (WebView2)** option to enable WebView2 script debugging.  <!--More details to follow soon.  -->  
+
+:::image type="complex" source="../media/vs-script-debugger.jpg" alt-text="Visual Studio JavaScript Debugger":::
+   Visual Studio JavaScript Debugger
+:::image-end:::
+
+### Visual Studio Code  
+
+Use Visual Studio Code to debug your script within the WebView2 right from the IDE.  For more details, see [Microsoft Edge(Chromium) WebView Application](https://github.com/microsoft/vscode-edge-debug2/blob/master/README.md#microsoft-edge-chromium-webview-applications) section of README on GitHub.
```
Comment (Reezaali) [inline microsoft-edge/webview2/howto/debug.md:46]:
```suggestion
You may use Visual Studio Code to debug scripts that run in WebView2 controls.  For more information, see [Microsoft Edge (Chromium) WebView Applications](https://github.com/microsoft/vscode-edge-debug2/blob/master/README.md#microsoft-edge-chromium-webview-applications).  
```

File: microsoft-edge/webview2/howto/debug.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+:::image type="complex" source="../media/vs-script-debugger.jpg" alt-text="Visual Studio JavaScript Debugger":::
+   Visual Studio JavaScript Debugger
+:::image-end:::
+
+### Visual Studio Code  
+
+Use Visual Studio Code to debug your script within the WebView2 right from the IDE.  For more details, see [Microsoft Edge(Chromium) WebView Application](https://github.com/microsoft/vscode-edge-debug2/blob/master/README.md#microsoft-edge-chromium-webview-applications) section of README on GitHub.  
+
+## Getting in tough with the Microsoft Edge WebView team
```
Comment (Reezaali) [inline microsoft-edge/webview2/howto/debug.md:48]:
```suggestion
## Getting in touch with the Microsoft Edge WebView team  
```

File: microsoft-edge/webview2/howto/debug.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   Visual Studio JavaScript Debugger
+:::image-end:::
+
+### Visual Studio Code  
+
+Use Visual Studio Code to debug your script within the WebView2 right from the IDE.  For more details, see [Microsoft Edge(Chromium) WebView Application](https://github.com/microsoft/vscode-edge-debug2/blob/master/README.md#microsoft-edge-chromium-webview-applications) section of README on GitHub.  
+
+## Getting in tough with the Microsoft Edge WebView team  
+
+Help build a richer WebView2 experience by sharing your feedback!  Visit the [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports.
```
Comment (Reezaali) [inline microsoft-edge/webview2/howto/debug.md:50]:
```suggestion
Help build a richer WebView2 experience by sharing your feedback.  Visit the [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports.  
```

File: microsoft-edge/webview2/howto/debug.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+### Visual Studio Code  
+
+Use Visual Studio Code to debug your script within the WebView2 right from the IDE.  For more details, see [Microsoft Edge(Chromium) WebView Application](https://github.com/microsoft/vscode-edge-debug2/blob/master/README.md#microsoft-edge-chromium-webview-applications) section of README on GitHub.  
+
+## Getting in tough with the Microsoft Edge WebView team  
+
+Help build a richer WebView2 experience by sharing your feedback!  Visit the [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports.  
+
+## See also
```
Comment (Reezaali) [inline microsoft-edge/webview2/howto/debug.md:52]: Please add a couple of links. Thanks.

Comment (jm-trd-ms) [reply microsoft-edge/webview2/howto/debug.md]: Added note inline.

File: microsoft-edge/webview2/howto/debug.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+> [!NOTE]
+When you debug your application in Visual Studio with the native debugger attached, pressing `F12` may trigger the native debugger instead of Developer Tools.  Use `Ctrl`+`Shift`+`I`, or right-click and select `Inspect` to avoid potential hotkey conflict.  
+
+### Visual Studio  
+
+Use the script debugger in Visual Studio 2019 version 16.4 Preview 2 or later to debug your script within WebView2 right from the IDE.  Make sure the **JavaScript diagnostics** component in **Desktop development with C++** workload is installed.  
+
+:::image type="complex" source="../media/vs-js-diagnostics.jpg" alt-text="Visual Studio JavaScript diagnostics":::
+   Visual Studio JavaScript diagnostics
+:::image-end:::
```
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/webview2/howto/debug.md:38]: Added note inline.

File: microsoft-edge/webview2/howto/debug.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+Right click on your project and select **Properties**.  Under **Configuration Properties**, select **Debugging** > **Debugger Type**, and select the **JavaScript (WebView2)** option to enable WebView2 script debugging.  <!--More details to follow soon.  -->  
+
+:::image type="complex" source="../media/vs-script-debugger.jpg" alt-text="Visual Studio JavaScript Debugger":::
+   Visual Studio JavaScript Debugger
+:::image-end:::
```
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/webview2/howto/debug.md:44]: Added note inline.

File: microsoft-edge/webview2/howto/debug.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+## Microsoft Edge DevTools  
+
+### DevTools  
+
+Use [Microsoft Edge (Chromium) Developer Tools](/microsoft-edge/devtools-guide-chromium) to debug web content displayed in WebView, in the same way as Microsoft Edge.  Focus on the WebView window, press `F12`, or press `Ctrl`+`Shift`+`I`, or right-click and select `Inspect` to open Developer Tools.  
+
+:::image type="complex" source="../media/f12.PNG" alt-text="F12":::
+   F12
+:::image-end:::
```
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/webview2/howto/debug.md]: Updated.

File: microsoft-edge/webview2/howto/debug.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+> [!NOTE]
+When you debug your application in Visual Studio with the native debugger attached, pressing `F12` may trigger the native debugger instead of Developer Tools.  Use `Ctrl`+`Shift`+`I`, or right-click and select `Inspect` to avoid potential hotkey conflict.  
+
+### Visual Studio  
+
+Use the script debugger in Visual Studio 2019 version 16.4 Preview 2 or later to debug your script within WebView2 right from the IDE.  Make sure the **JavaScript diagnostics** component in **Desktop development with C++** workload is installed.
```
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/webview2/howto/debug.md]: Updated

**Comment by jm-trd-ms** [comment]
#signed-off and merged.

**Pull request merged**